snakemake.el

```diff
@@ -350,6 +350,17 @@ targets."
                                        (snakemake-all-rules))
              " "))
 
+(defun snakemake-graph-workflow ()
+  (interactive)
+  """Creates a graph for the current snakefile."""
+  (let ((dot (shell-command-to-string "which dot")))
+    (when (equal dot "")
+      (error "You do not have dot (graphviz) on your emacs path!")))
+  (let* ((fname (buffer-file-name))
+         (graphname (concat (buffer-file-name) ".pdf"))
+         (graph-command (concat "snakemake --dag --snakefile " fname " | dot -Tpdf > " graphname)))
```

Comment:
Hmm, but this is limited to creating the graph for the first rule of
the Snakefile, right?  Instead, a compile command could be constructed
(similar to what is done in `snakemake-compile`).  This could default
to naming the output something like `<rule>-dag.pdf`, but the user could
edit the compile command further.